tutorcredentials/__about__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "16.0.2"
+__version__ = "16.0.3"
```

Comment:
Please don't bump the version number here! I'll recap the process:

- Open a first PR with just your changes:
    - Include just your operating changes
    - Create a changelog entry with `scriv create`
    - Discuss until your PR is merged
- If you want to publish a new version, then open a new PR:
    - bump the version number in `__about__.py`
    - collect changelog entries with `scriv collect`
    - `git commit -a -m v16.xyz`
    - Open a new PR. You can merge that PR without approval, since it does not contain any major change.

If you bump the version number in the first PR, then other committers will have git conflicts. It will also be more difficult to cherry-pick changes in other release branches.